scripts/run-clang-tidy.py

```diff
@@ -82,6 +82,7 @@ def get_tidy_invocation(f, clang_tidy_binary, checks, tmpdir, build_path,
                         config):
     """Gets a command line for clang-tidy."""
     start = [clang_tidy_binary]
+    start.append('-warnings-as-errors=*')
```

Comment:
nit: later let's change this to `run_clang_tidy.py`

Reply:
Agreed!